hw/formal/tools/jaspergold/fpv.tcl

```diff
@@ -83,7 +83,7 @@ if {[info exists ::env(AFTER_LOAD)]} {
 # tlul_assert.sv operates on the negedge clock
 # even clock this sampled at both_edges, input should only change at posedge clock cycle
 # TODO: create each DUT_TOP's individual config file
-if {$env(DUT_TOP) == "pinmux_tb"} {
+if {($env(DUT_TOP) == "pinmux_tb") || ($env(DUT_TOP) == "pinmux_chip_tb")} {
```

Comment:
@matutem this change is needed to also properly define the clocks and resets for the "new" prim_chip_tb target. Without this, FPV would quickly find hundreds of CEXs.